packages/edit-site/src/components/editor/index.js

```diff
@@ -212,6 +212,14 @@ export default function EditSiteEditor( { isPostsList = false } ) {
 		duration: disableMotion ? 0 : 0.2,
 	};
 
+	const { enableComplementaryArea } = useDispatch( interfaceStore );
+
+	useEffect( () => {
+		if ( path === '/wp_global_styles' && canvas === 'edit' ) {
+			enableComplementaryArea( 'core', 'edit-site/global-styles' );
+		}
+	}, [ path, enableComplementaryArea, canvas ] );
```

Comment:
Do you think this effect should move to the "GlobalStylesSidebar" component instead?

Reply:
We could. This component already seemed away of the path. Btw, maybe we should change it back to the document panel when moving away from `'/wp_global_styles'`? I'm not sure.

Reply:
(not entirely sure)

Reply:
Right now it will stay open if you go to another editor, such as a page under "Pages".

Reply:
oh good catch :)

Reply:
I moved the effect as you suggested and also restore the previous state now.